src/SprykerEco/Zed/Heidelpay/Dependency/Service/HeidelpayToUtilEncodingBridge.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Heidelpay\Dependency\Service;
+
+class HeidelpayToUtilEncodingBridge implements HeidelpayToUtilEncodingInterface
```

Comment:
Please note that these according to new standards should always include the type: 
...ServiceBridge and ...ServiceInterface

Reply:
@mykyta65536 this has the lowest prio